src/averageAges.js

```diff
@@ -38,6 +52,26 @@ function calculateMenAverageAge(people, century) {
  */
 function calculateWomenAverageAge(people, withChildren) {
   // write code here
+  const filteredWomen = people.filter((person) => person.sex === 'f');
+  const filteredMothers = [];
+
+  for (const key of filteredWomen) {
```

Comment:
It`s not a key, it`s certain woman actually
```suggestion
  for (const woman of filteredWomen) {
```

Reply:
come on  i am not like that !!! 🤣🤣🤣🤣🤣🤣🤣 i respect each and every living thing !!!